modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/externalize/RelJsonReaderTest.java

```diff
@@ -71,5 +77,49 @@ void fromJson() {
         assertThat(node.getTable(), notNullValue());
         assertThat(node.getTable().unwrap(IgniteTable.class), is(igniteTableMock));
         Mockito.verify(schemaMock).tableById(tableId);
+
+        AtomicBoolean tableByIdCalled = new AtomicBoolean();
+        RelJsonReader reader = new RelJsonReader(schemaMock);
+
+        when(schemaMock.tableById(any())).thenAnswer((k) -> {
+            tableByIdCalled.set(true);
+
+            return igniteTableMock;
+        });
+
+        reader.read(json);
+        assertTrue(tableByIdCalled.get());
+    }
+
+    /**
+     * Checks that appropriate methods called form table manager on rel node construction.
+     */
+    @Test
+    void checkTableManagerCalled() throws Exception {
```

Comment:
This doesn't look right, because we use JsonReader to validate behaviour of SqlSchemaManager. Let's move this test to distinct class and remove all deserialisation related code

Reply:
plz check, is it ok for now?